holidayextras/brands
# Improve essential travel navbar

#### What does this PR do? (please provide any background)
- The essential travel overbrand was done in isolation and when introducing it to pattern library we noticed it doesn't work quite as expected. There is no visible logo. and it struggles on mobile.
- This PR adjusts the navbar styling so that a logo is visible. We have also created a new essential travel navbar in patter library to see how this works (branch the same name)

#### What tests does this PR have?
None

#### How can this be tested?
Checkout this branch on both this repo `brands` and pattern library
in pattern library run `npm link ../brands` then run `npm start`
Navigate to organisms > Essential Navbar and check how this works across device


#### Screenshots / Screencast
<img width="1198" alt="screen shot 2016-02-25 at 17 12 46" src="https://cloud.githubusercontent.com/assets/778942/13327866/9f151616-dbe3-11e5-94a8-8c6077c9fe99.png">
<img width="440" alt="screen shot 2016-02-25 at 17 13 03" src="https://cloud.githubusercontent.com/assets/778942/13327865/9f123a4a-dbe3-11e5-8410-cdb57401d598.png">



#### What gif best describes how you feel about this work?
![Blink 182](https://media.giphy.com/media/aZ3LDBs1ExsE8/giphy.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
